tests/zfs-tests/tests/functional/mmp/zpool_import_mmp_test_template.ksh

```diff
@@ -0,0 +1,50 @@
+#!/bin/ksh -p
```

Comment:
I think we no longer need this template since any other mmp test can be used as a template, right?